src/analyze_includes/evaluate_includes.py

```diff
@@ -70,7 +70,7 @@ def to_json(self) -> str:
         return dumps(content, indent=2) + "\n"
 
     @staticmethod
-    def _make_includes_map(includes: List[Include]) -> DefaultDict[Any, List[str]]:
+    def _make_includes_map(includes: List[Include]) -> DefaultDict[str, List[str]]:
```

Comment:
Not sure if this change makes it more clear...

Reply:
The mypy integration of my IDE claims it has to be `Any`. Then again, we are explicitly construction every entry with a string key. It is indeed confusing. If we ever introduce mypy we can for sure somehow silent this line.